src/widgets/types.ts

```diff
@@ -247,3 +247,7 @@ type DisplayMetadata = {
   links?: Links;
   uuid?: string;
 };
+
+export interface widgetBasePath {
```

Comment:
I think this is really confusing for a type name. My mind constantly sees this as a variable when imported. Why not call this `WidgetBasePath` following the other patterns in the file?